ansible/pbTestScripts/vmDestroy.sh

```diff
@@ -1,50 +1,82 @@
 #!/bin/bash
 set -eu
 
+osToDestroy=''
+force=False
 # Takes in all arguments
 processArgs()
 {
-	if [ ! -n "${WORKSPACE:-}" ]; then
-		echo "WORKSPACE not found, setting it as environment variable 'HOME'"
-		WORKSPACE=$HOME
-	fi
-	if [ $# -lt 1 ]; then
-		echo "Script takes 1 input argument: "
-		echo "The project whose VMs are to be destroyed"
-		exit 1
-	fi
-	
+	while [[ $# -gt 0 ]] && [[ ."$1" = .-* ]] ; do
+		local opt="$1";
+		shift;
+		case "$opt" in
+			"--OS" | "-o" )
+				osToDestroy=$1; shift;;
+			"--force" | "-f" )
+				force=True;;
+			"--help" | "-h" )
+				usage; exit 0;;
+			*) echo >&2 "Invalid option: ${opt}"; echo "This option was unrecognised."; usage; exit 1;;
+		esac
+	done
 }
 
-# Takes project name as arg 1, and OS as arg 2
-destroyVM()
-{
-	cd $WORKSPACE/adoptopenjdkPBTests/$1/ansible
-	ln -sf Vagrantfile.$2 Vagrantfile	# Correct Vagrantfile alias
-	vagrant destroy -f			# Force destroy without question
+usage() {
+
+		echo "Usage: ./vmDestroy <option>
```

Comment:
Add `-o <os_list>` to this message, and indicate that the list is space separated underneath

Reply:
Also it's `./vmDestroy.sh`